src/Entity/User.php

```diff
@@ -93,7 +93,7 @@ public function setEmail(string $email)
      */
     public function setPassword(string $password)
     {
-        $this->password = $password;
+        $this->password = md5($password);
```

Comment:
move this out of the setter. it is very counterintuitive, and you might end up md5-ing an md5

Reply:
also, the entity should not care about the hashing. you should have a service that deals with password hashing.